src/site/stages/build/drupal/graphql/vetCenter.graphql.js

```diff
@@ -0,0 +1,83 @@
+const vetCenterFragment = `
+ fragment vetCenterFragment on NodeVetCenter {
+        entityId
+        entityUrl {
+          path
+          routed
+        }
+        entityBundle
+        entityLabel
+        fieldIntroText
+        fieldOfficeHours {
+          day
+          starthours
+          endhours
+          comment
+        }
+        # Other locations link:
+        reverseFieldOfficeNode(limit: 500, filter:{conditions: [{field: "type", value: ["vet_center_locations_list"]}]}) {
+          entities {
+            ... on NodeVetCenterLocationsList {
+              entityUrl {
+                path
+                routed
+              }
+            }
+          }
+        }        
+        fieldPrepareForVisit {
+          entity {
+            ... on ParagraphBasicAccordion {
+              fieldHeader
+              fieldRichWysiwyg {
+                processed
+              }
+            }
+          }
+        }        
+        fieldHealthServices {
+          entity {
+            ... on NodeVetCenterFacilityHealthServi {
+              fieldServiceNameAndDescripti {
+                entity {
+                  ... on TaxonomyTermHealthCareServiceTaxonomy {
+                    name
+                    fieldVetCenterTypeOfCare
+                    fieldServiceTypeOfCare
+                    fieldVetCenterFriendlyName
+                    fieldAlsoKnownAs
+                    fieldVetCenterComConditions
+                    fieldCommonlyTreatedCondition
+                    fieldVetCenterServiceDescrip
+                    description {
+                      processed
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }`;
+
+const GetVetCenters = `
+  ${vetCenterFragment}
+  
+  query GetVetCenters {
+    nodeQuery(limit: 1000, filter: {
+      conditions: [
+        { field: "status", value: ["1"], enabled: $onlyPublishedContent },      
+        { field: "type", value: ["vet_center"] }
+      ]
+    }) {
+      entities {
+        ... vetCenterFragment
+      }
+    }
+  }
+`;
+
+module.exports = {
+  fragment: vetCenterFragment,
+  GetVetCenters,
+};
```

Comment:
If there is going to be a lot of instances of these, you may want to implement or write a ticket for implementing pagination. See `src/site/stages/build/drupal/graphql/vaFormPage.graphql.js` for an example